quick_tour/flex_recipes.rst

```diff
@@ -0,0 +1,249 @@
+Composer plugin that is already installed in our project.
+
+First, ``twig`` is not the name of a Composer package: it's a Flex *alias* that
+points to ``symfony/twig-bundle``. Flex resolves that alias for Composer.
+
+And second, Flex installs a *recipe* for ``symfony/twig-bundle``. What's a recipe?
+It's a way for a library to automatically configure itself by adding and modifying
+files. Thanks to recipes, adding features is seamless and automated: install a package
+and you're done!
+
+You an find a full list of recipes and aliases by going to `https://symfony.sh`_.
+
+What did this recipe do? In addition to automatically enabling the bundle in
+``config/bundles.php``, it added 3 things:
+
+``config/packages/twig.yaml``
+    A configuration file that sets up Twig with sensible defaults.
+
+``config/routes/dev/twig.yaml``
+    A route that helps you debug your error pages: it's added only in the ``dev``
+    environment.
+
+``templates/``
+    This is the directory where template files will live. The recipe also added
+    a ``base.html.twig`` layout file.
+
+Twig: Rendering a Template
+--------------------------
+
+Thanks to Flex, after one command, you can start using Twig immediately::
+
+    // src/Controller/DefaultController.php
+    // ...
+
+    + use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+ 
+    -class DefaultController
+    +class DefaultController extends AbstractController
+     {
+         /**
+          * @Route("/hello/{name}")
+          */
+         public function index($name)
+         {
+    -        return new Response("Hello $name!");
+    +        return $this->render('default/index.html.twig', [
+    +            'name' => $name,
+    +        ]);
+         }
+
+By extending ``AbstractController``, you now have access to a number of shortcut
+methods and tools, like ``render()``. Create the new template:
+
+.. code-block:: twig
+
+    {# templates/default/index.html.twig #}
+    <h1>Hello {{ name }}</h1>
+
+That's it! The ``{{ name }}`` syntax will print the ``name`` variable that's passed
+in from the controller. If you're new to Twig, welcome! You'll learn more about
+its syntax and power later.
+
+But, right now, the page *only* contains the ``h1`` tag. To give it an HTML layout,
+extend ``base.html.twig``:
+
+.. code-block:: twig
+
+    {# templates/default/index.html.twig #}
+    {% extends 'base.html.twig' %}
+
+    {% block body %}
+        <h1>Hello {{ name }}</h1>
+    {% endblock %}
+
+This is called template inheritance: our page now inherits the HTML structure from
+``base.html.twig``.
+
+Profiler: Debugging Paradise
+----------------------------
+
+One of the *coolest* features of Symfony isn't even installed yet! Let's fix that:
+
+.. code-block:: terminal
+
+    $ composer require profiler
+
+Yes! This is another alias! And Flex *also* installs another recipe, which automates
+the configuration of Symfony's WebProfilerBundle. What's the result? Refresh!
```

Comment:
I'd prefer to never mention bundles in the Quick tour. What if we replace `Symfony's WebProfilerBundle` by `Symfony's Profiler` ?